src/splints.iml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
```

Comment:
(L) I gather this belongs to a partial IntelliJ build system. It should not be in `src/`; recall your "src" is really under `src/java/`